ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -15,12 +33,16 @@ tests:
 - as: unit-tests
   commands: ci/prow/unit-tests
   container:
-    from: src
+    clone: true
+    from: ocp_builder_rhel-8-golang-1.18-openshift-4.12
 - as: lint
-  commands: ci/prow/lint
+  commands: |
+    export GOCACHE=/tmp
+    export GOLANGCI_LINT_CACHE=/tmp
+    make lint GOLANGCI_LINT=/usr/bin/golangci-lint
```

Comment:
Why don't we export `GOLANGCI_LINT` as well?
I guess that it means we also need to update `ci/prow/lint` and change the command to `ci/prow/lint` right?!

Reply:
`GOLANGCI_LINT=/usr/bin/golangci-lint ci/prow/lint` does not work because `make` does not import environment variables into its own variables, for some reason.
The change to `ci/prow/lint` is in [that PR](https://github.com/rh-ecosystem-edge/kernel-module-management/pull/14#event-7169694405). PTAL.